eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -141,24 +141,24 @@ class Peer(val nodeParams: NodeParams, remoteNodeId: PublicKey, wallet: OnChainA
+        if (c.fundingAmount >= Channel.MAX_FUNDING && !d.localFeatures.hasFeature(Wumbo)) {
+          sender() ! Status.Failure(new RuntimeException(s"fundingAmount=${c.fundingAmount} is too big, you must enable large channels support in 'eclair.features' to use funding above ${Channel.MAX_FUNDING} (see eclair.conf)"))
           stay()
-        } else if (c.fundingSatoshis >= Channel.MAX_FUNDING && !d.remoteFeatures.hasFeature(Wumbo)) {
-          sender() ! Status.Failure(new RuntimeException(s"fundingSatoshis=${c.fundingSatoshis} is too big, the remote peer doesn't support wumbo"))
+        } else if (c.fundingAmount >= Channel.MAX_FUNDING && !d.remoteFeatures.hasFeature(Wumbo)) {
+          sender() ! Status.Failure(new RuntimeException(s"fundingAmount=${c.fundingAmount} is too big, the remote peer doesn't support wumbo"))
           stay()
-        } else if (c.fundingSatoshis > nodeParams.channelConf.maxFundingSatoshis) {
-          sender() ! Status.Failure(new RuntimeException(s"fundingSatoshis=${c.fundingSatoshis} is too big for the current settings, increase 'eclair.max-funding-satoshis' (see eclair.conf)"))
+        } else if (c.fundingAmount > nodeParams.channelConf.maxFundingSatoshis) {
+          sender() ! Status.Failure(new RuntimeException(s"fundingAmount=${c.fundingAmount} is too big for the current settings, increase 'eclair.max-funding-satoshis' (see eclair.conf)"))
           stay()
         } else {
           // If a channel type was provided, we directly use it instead of computing it based on local and remote features.
-          val channelFlags = c.channelFlags.getOrElse(nodeParams.channelConf.channelFlags)
-          val channelType = c.channelType_opt.getOrElse(ChannelTypes.defaultFromFeatures(d.localFeatures, d.remoteFeatures, announceChannel = channelFlags.announceChannel))
+          val channelFlags = c.channelFlags_opt.getOrElse(nodeParams.channelConf.channelFlags)
+          val channelType = c.channelType_opt.getOrElse(ChannelTypes.defaultFromFeatures(d.localFeatures, d.remoteFeatures, channelFlags.announceChannel))
           // NB: we need to capture parameters in a val to use them in andThen
           val selfRef = self
           val origin = sender()
           implicit val ec: ExecutionContext = ExecutionContext.Implicits.global
-          createLocalParams(nodeParams, d.localFeatures, channelType, isInitiator = true, c.fundingSatoshis).andThen {
+          createLocalParams(nodeParams, d.localFeatures, channelType, isInitiator = true, c.fundingAmount).andThen {
             case Success(localParams) => selfRef ! SpawnChannelInitiator(c, ChannelConfig.standard, channelType, localParams, origin)
             case Failure(t) => origin ! Status.Failure(new RuntimeException("channel creation failed", t))
```

Comment:
Nit (to preserve sender):
```suggestion
            case Failure(t) => origin.tell(Status.Failure(new RuntimeException("channel creation failed", t)), selfRef)
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/2247/commits/31c3ee911cca907b62b82f5bc8ecedcaa1ad0563